template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Solution/Solution1.sln

```diff
@@ -1,8 +1,8 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26124.0
-MinimumVisualStudioVersion = 15.0.26124.0
+# Visual Studio Version 16
+VisualStudioVersion = 16.6.30114.105
+MinimumVisualStudioVersion = 16.6.30114.105
```

Comment:
The minimum version required is 16.6 as documented in [Create a .NET Core console application using Visual Studio](https://docs.microsoft.com/en-us/dotnet/core/tutorials/with-visual-studio).

Reply:
@NextTurn the documentation page says that the provided instructions require VS minimum version 2019 16.6. The MinimumVisualStudioVersion should be 10.0.40219.1, otherwise created from CLI `*.sln` file could not be opened by other versions and this would break backward compatibility.